integrations/apollo/src/index.ts

```diff
@@ -24,7 +25,8 @@ export type NhostApolloClientOptions = {
   connectToDevTools?: boolean
   cache?: InMemoryCache
   onError?: RequestHandler
-  link?: ApolloClient<any>['link']
+  link?: ApolloLink
+  generateLinks: (links: (ApolloLink | RequestHandler)[]) => (ApolloLink | RequestHandler)[]
```

Comment:
```suggestion
  generateLinks?: (links: (ApolloLink | RequestHandler)[]) => (ApolloLink | RequestHandler)[]
```

I think we shouldn't necessarily force everyone to use it. Also, I'm not sure if this really needs to be a function and not just simply `(ApolloLink | RequestHandler)[]` instead.

Reply:
Yes, that's exactly the use case.

It's nice to be able to reuse the httplink and wslink that nhost manages for token renewal etc. But the order and all other links are nice to have full control over.